goss-testing/tests/ncn/goss-k8s-istio-pods-running.yaml

```diff
@@ -1,14 +1,34 @@
-# Copyright 2014-2021 Hewlett Packard Enterprise Development LP
-# Variable set: pods_istio
+# MIT License
+#
+# (C) Copyright [2021-2022] Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
 command:
-  {{range $label := index .Vars "pods_istio"}}
```

Comment:
Is the pods_istio used anywhere at this point.  If not, should the vars also be cleaned up?
https://github.com/Cray-HPE/csm-testing/blob/master/goss-testing/vars/variables-ncn.yaml#L51

Reply:
Brad and I decided not to change any thing with regards to the old suite and this before the 1.2 release goes out.